Radzen.Blazor/RadzenDayView.razor

```diff
@@ -7,13 +7,54 @@
     public override RenderFragment Render()
     {
         var appointments = Scheduler.GetAppointmentsInRange(StartDate, EndDate).ToList();
+        var resources = Scheduler.ActiveResourceList;
 
-        return
-    @<CascadingValue Value=@Scheduler>
-        <DayView StartDate=@StartDate EndDate=@EndDate StartTime=@StartTime EndTime=@EndTime Appointments=@appointments
-            TimeFormat=@TimeFormat MinutesPerSlot=@MinutesPerSlot
-            AppointmentMove=OnAppointmentMove />
-    </CascadingValue>
-    ;
+        if (resources.Count > 0 && ShowResourceGrouping)
+        {
+            Dictionary<ISchedulerResource, ISchedulerResource> resourceDictionary;
+            resourceDictionary = new();
+            int count = resources.Count;
+            int next = 1;
+            foreach (var resource in resources)
+            {
+                if (next < count)
+                {
+                    resourceDictionary[resource] = resources[next];
+                }
+                else
+                {
+                    resourceDictionary[resource] = null;
+                }
+                next++;
+            }
+            var currentResource = resourceDictionary.FirstOrDefault().Key;
+
+            return
+                @<div class="rz-resource-scheduler">
```

Comment:
Why is RadzenDayView rendering the resource header? I think RadzeResourceDayView should do that and RadzenDayView should stay mostly untouched. The same goes for the other views.

For example I think RadzenResourceDayView should:

1. Render the resource headers
2. Render RadzenDayView for every resource.